src/components/SquareInfoBox.tsx

```diff
@@ -1,6 +1,69 @@
-import React from 'react'
+import React, {CSSProperties} from 'react'
+import { Box } from '@material-ui/core'
 
-export function SquareInfoBox(props) {
+const boxStyle: CSSProperties = {
```

Comment:
We are using `styled` instead of defining the CSS. Check the other components to see how it's being done, also, this comment from Jordan https://github.com/dOrgTech/landing-page-v2/pull/9#discussion_r544743818 might be useful too

Reply:
I updated the file to use the material ui styling API with styled()